internal/config/settings.go

```diff
@@ -315,6 +315,7 @@ var defaultCommonSettings = map[string]interface{}{
 	"tabstospaces":   false,
 	"useprimary":     true,
 	"wordwrap":       false,
+	"altlinecolors":  false,
```

Comment:
These lines are alphabetically sorted, so `altlinecolors` should be added not to the very end of `defaultCommonSettings` but to the very beginning of it. :)